src/simtools/simtel/simtel_config_writer.py

```diff
@@ -422,3 +439,81 @@ def _get_array_triggers_for_telescope_type(self, array_triggers, telescope_type)
             if trigger_dict["name"] == telescope_type + "_array":
                 return trigger_dict
         return None
+
+    def write_dummy_telescope_configuration_file(
+        self, parameters, config_file_path, telescope_name
+    ):
+        """
+        Write 'dummy' telescope configuration file used as zeroth telescope in sim_telarray.
+
+        Replaces key telescope configuration values with dummy values.
+
+        Parameters
+        ----------
+        parameters: dict
+            Telescope parameters used as template.
+        config_file_path: str or Path
+            Path of the dummy configuration file to write on.
+        telescope_name: str
+            Name of the telescope.
+        """
+        self._logger.debug(f"Writing {telescope_name} telescope config file {config_file_path}")
+        defaults = {
+            "camera_config_file": f"{telescope_name}_single_pixel_camera.dat",
+            "discriminator_pulse_shape": f"{telescope_name}_pulse.dat",
+            "fadc_pulse_shape": f"{telescope_name}_pulse.dat",
+            "mirror_list": f"{telescope_name}_single_12m_mirror.dat",
+            "mirror_reflectivity": f"{telescope_name}_reflectivity.dat",
+            "camera_pixels": 1,
+            "trigger_pixels": 1,
+            "num_gains": 1,
+            "fadc_bins": 10,
+            "disc_bins": 10,
+            "fadc_sum_bins": 10,
+            "fadc_sum_offset": 0,
+            "asum_threshold": 0,
+            "dsum_threshold": 0,
+            "discriminator_threshold": 1,
+            "fadc_amplitude": 1.0,
+            "discriminator_amplitude": 1.0,
+        }
+
+        for key, val in defaults.items():
+            if key in parameters:
+                parameters[key]["value"] = val
+
+        self.write_telescope_config_file(config_file_path, parameters, telescope_name, True)
+
+        config_file_directory = Path(config_file_path).parent
+        self._write_dummy_mirror_list_file(config_file_directory, telescope_name)
+        self._write_dummy_camera_file(config_file_directory, telescope_name)
```

Comment:
I recommend adding `s` to these functions so it's clear it writes multiple relevant files.